File: ChatTemplate/ClientApp/src/app/models/cell.ts
import { CellStatus } from './cell-status.enum';

export class Cell {

  public x: number;
  public y: number;
  public status: CellStatus;

  constructor(x: number, y: number) {
    this.x = x;
    this.y = y;
    this.status = CellStatus.clear;
  }
}

File: ChatTemplate/Models/Cell.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ChatTemplate.Models
{
    public class Cell
    {
        public int x { get; set; }
        public int y { get; set; }
        public CellStatus status { get; set; }
    }
}
